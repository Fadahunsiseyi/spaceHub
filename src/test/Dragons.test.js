import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import DragonList from '../pages/Dragons';

describe('Run the test for the reserved list of dragons', () => {
  it('To Test Dragons component renders successfully!', () => {
    const app = renderer.create(
      <Provider store={store}>
        <DragonList />
      </Provider>,
    ).toJSON();
    expect(app).toMatchSnapshot();
  });
});
