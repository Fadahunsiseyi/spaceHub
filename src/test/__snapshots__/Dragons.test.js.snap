// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Run the test for the reserved list of dragons To Test Dragons component renders successfully! 1`] = `
<div
  className="rockets"
/>
`;
